### ISSUE TITLE
Renko Price Transformation

### ISSUE THREAD

**Issue by vishalsmart** [DaveSkender/Stock.Indicators#469]
Need a feature -
1. Renko Price Transformation
2. Other indicators need to use Renko price to generate, i.e. supertrend, rsi etc need to follow renko transformation.

**Comment by DaveSkender** [comment]
@vishalsmart is this an accurate depiction of what you're seeking:
https://www.investopedia.com/terms/r/renkochart.asp

Also, can you elaborate on "need to" in #2.  I suspect you are interested in doing a SuperTrend or RSI using Renko prices, right?  And not that you are implying that standard SuperTrend and RSI should use it.

**Comment by vishalsmart** [comment]
@DaveSkender yes, I am referring for the Renko bricks like you have mentioned. Like you have HeikenAshi transformation, it should be like that. But, in Renko bricks, it will not be same number of output.

Here is something which is converting Ticks to Bricks -
https://github.com/squideyes/RenkoFeedDemo/tree/master/RenkoFeedDemo

I think need a parameter to input brick size while applying transformation.

**Comment by DaveSkender** [comment]
Ok, thanks for the info.  I'll probably make a `.ConvertToQuotes()` extension for Renko, HeikinAshi, and maybe some others to make it easier to use as a source for other indicators, something like:

```csharp
var results = history.GetRenko()
  .ConvertToQuotes()
  .GetRsi(14);
```

**Comment by vishalsmart** [comment]
Really appreciated, but in GetRenko() pls make to get custom brick size like:

var results = history.GetRenko(10)
  .ConvertToQuotes()
  .GetRsi(14);

**Comment by DaveSkender** [comment]
Will do.  There will also be a parameter to specify different variants as well (High/Low vs Close; ATR or Fixed sized bricks, etc).

**Comment by vishalsmart** [comment]
@DaveSkender thank you very much. Really appreciated.

**Comment by DaveSkender** [comment]
This was released in version 1.14.0 as a Preview.  Comments and feedback are appreciated.
